vault/token_store.go

```diff
@@ -2173,7 +2173,7 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(
 			}
 			resp.AddWarning(fmt.Sprintf(
 				"Given explicit max TTL of %d is greater than system/mount allowed value of %d seconds; until this is fixed attempting to create tokens against this role will result in an error",
-				entry.ExplicitMaxTTL.Seconds(), sysView.MaxLeaseTTL().Seconds()))
+				int(entry.ExplicitMaxTTL.Seconds()), sysView.MaxLeaseTTL().Seconds()))
```

Comment:
This should be an `int64`, and you only `int64`'d the first parameter -- both should be.